ci-operator/step-registry/cucushift/installer/rehearse/ibmcloud/ipi/minimal-permission/deprovision/cucushift-installer-rehearse-ibmcloud-ipi-minimal-permission-deprovision-chain.yaml

```diff
@@ -0,0 +1,6 @@
+chain:
+  as: cucushift-installer-rehearse-ibmcloud-ipi-minimal-permission-deprovision
+  steps:
+  - chain: cucushift-installer-rehearse-ibmcloud-ipi-deprovision
```

Comment:
do we need dedicated steps to remove the created service id and service key?

Reply:
yes, I found out the service id is left after destroyed the cluster.